scripts/install

```diff
@@ -43,7 +44,7 @@ test ! -e "$final_path" || ynh_die --message="This path already contains a folde
 ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
 # Check memory requirements
-check_memory_requirements
+#check_memory_requirements
```

Comment:
This should be uncommented if we want to avoid failing install/functioning by lack of memory

Reply:
I'm guessing this is to work around the issue on the CI where job fails because of no swap ... but we could wrap this in an `if $PACKAGE_CHECK` (i never remember what's the proper variable name)

Reply:
I just added this to the scripts
```
if [ ${PACKAGE_CHECK_EXEC:-0} -eq 1 ]; then
	:
else
	# Check memory requirements
	check_memory_requirements
fi
```